golem-cluster.yaml

```diff
@@ -100,7 +101,9 @@ available_node_types:
     # The node type's CPU and GPU resources
     resources: {}
 
-    node_config: {} # TODO: Demand description here
+    # Additional parameters specific for this node type added on top of node_config from provider.parameters.node_config
+    node_config: {}
```

Comment:
Can we add an example `yaml` that is showcasing usage of different configuration for both stacks?